### PR TITLE
[O2B-645] Include tags update in run update

### PR DESCRIPTION
**Pull request by martinboulais**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [ ] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [ ] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

### CODE REVIEW

File: lib/usecases/run/UpdateRunUseCase.js [hunk 1]
```diff
@@ -126,6 +129,24 @@ class UpdateRunUseCase {
                 if (eorReasons) {
                     await this._updateEorReasonsOnRun(runId, eorReasons);
                 }
+                if (tags) {
+                    const existingTags = await new GetAllTagsUseCase()
+                        .execute({ query: { page: { filter: { ids: tags } } } });
+                    const missingTags = tags.reduce((accumulator, tag) => {
+                        if (!existingTags.tags.find(({ id }) => id === tag)) {
+                            accumulator.push(tag);
```
Comment (JikHuijberts) [inline lib/usecases/run/UpdateRunUseCase.js:137]: The tests do not cover invalid tags yet.

Comment (martinboulais) [reply lib/usecases/run/UpdateRunUseCase.js:137]: This is the first step of a refactoring, I add test in the next PR

Comment (JikHuijberts) [reply lib/usecases/run/UpdateRunUseCase.js:137]: If the tests work on the next branch locally it is not a problem.

Comment (martinboulais) [reply lib/usecases/run/UpdateRunUseCase.js:137]: Yes, hey pass locally for the next branch

**Review by JikHuijberts — APPROVED** [review]

**Pull request merged**